Change /dev max name length from 40 (39) chars to 1024 (1023)
Patch author: Ryan Finnie <ryan@finnie.org>

Permanent mappings in /dev can become quite long, for example:

/dev/serial/by-id/usb-FTDI_FT232R_USB_UART_A800bZvc-if00-port0

This patch increases the maximum usable length to 1023 character.

Originally reported to Ubuntu:
    https://bugs.launchpad.net/bugs/978294

Will be sent upstream.

diff -ruN digitemp-3.6.0.dfsg.1.orig/src/digitemp.c digitemp-3.6.0.dfsg.1/src/digitemp.c
--- digitemp-3.6.0.dfsg.1.orig/src/digitemp.c	2008-08-27 21:42:48.000000000 -0700
+++ digitemp-3.6.0.dfsg.1/src/digitemp.c	2012-12-08 13:39:14.528173928 -0800
@@ -159,8 +159,8 @@
 
 extern const char dtlib[];			/* Library Used            */
  
-char serial_port[40],                        /* Path to the serial port */
-     tmp_serial_port[40], 
+char serial_port[1024],                        /* Path to the serial port */
+     tmp_serial_port[1024], 
      serial_dev[40],				/* Device name without /dev/ */
      log_file[1024],                         /* Path to the log file    */
      tmp_log_file[1024],
